fastlane/lib/fastlane/auto_complete.rb

```diff
@@ -0,0 +1,16 @@
+module Fastlane
+  # Enable tab auto completion
+  class AutoComplete
+    def self.execute
+      path = File.join(Fastlane::Helper.gem_path('fastlane'), 'lib', 'assets', 'completions')
+
+      fastlane_conf_dir = "~/.fastlane"
+      `mkdir -p #{fastlane_conf_dir} && cp -r #{path} #{fastlane_conf_dir}`
```

Comment:
+1 for switching to `FileUtils.mkdir_p` Make sure to use `File.expand_path`, since the `~` won't work otherwise